content/bluetooth.md

```diff
@@ -70,6 +71,19 @@ This process lowers the sound quality of the stream when in HSP/HFP mode, so aud
 
 ## Bluetooth Troubleshooting
 
+<!--## Dear Pop OS team I hope this helps to improve your distribution and documentation, feel free to rework.-->
+### Problem: Fresh installed pop-os_22.04_amd64_intel_38.iso, Bluetooth dont turn on
+The Bluetooth did not switch on in the settings. I can toggle it but it still stays gray.
+Solution:
+```bash
+sudo rmmod btusb
+sudo modprobe btusb
+```
+Now it works fine on my VivoBook.
+Source: https://www.reddit.com/r/pop_os/comments/mtjysf/bluetooth_wont_turn_on/
```

Comment:
We prefer to avoid links to outside websites, especially when they're semi-ephemeral like Reddit threads. If that thread gets deleted, we have a dead link in the docs.